adapters/improvedigital/params_test.go

```diff
@@ -45,9 +46,14 @@ var invalidParams = []string{
 	`[]`,
 	`true`,
 	`2`,
+	`{"size": {"w": 10, "h": 5}}`,
+	`{"other_optional": true}`,
```

Comment:
The fact that the above two entries were formerly valid but are not valid any more is concerning.  My impression is that this is a new adapter that may not yet have had any traffic, in which case this change is okay.  Can you please confirm that is the case?

Reply:
We don`t have PBS traffic. Just preparation and want to be strict on start.

"size" without "placementID" or (pkey+pubID) is invalid case, here is pb.js adapter https://github.com/prebid/Prebid.js/blob/master/modules/improvedigitalBidAdapter.js#L21 with the same logic.
Server also need placementID or (pkey+pubID), without them will be nobid/noad

"other_optional" also become invalid in case of bad configuration/typo/undocumented stuff 
Server will ignore "other_optional", but maybe somebody expect other result